apps/web/src/app/[orgShortcode]/[spaceShortcode]/settings/page.tsx

```diff
@@ -166,15 +166,20 @@ export default function SettingsPage() {
               </div>
             </div>
             <ColorField
-              initialValue={spaceSettings?.settings?.color}
-              showSaved={setShowSaved}
-              isSpaceAdmin={isSpaceAdmin}
-            />
-            <VisibilityField
-              initialValue={spaceSettings?.settings?.type}
+              initialValue={spaceSettings.settings.color}
               showSaved={setShowSaved}
               isSpaceAdmin={isSpaceAdmin}
             />
+            {/* Show visibility field only if the space is not a personal space or if it is a personal space and it is open so that users can change the visibility */}
```

Comment:
should always show the field
otherwise ux is confusing

Reply:
so just disable the open option on personal spaces?

Reply:
yes

Reply:
ok then the PR is already updated and ready